api_tests/btsieve/test.ts

```diff
@@ -419,7 +414,7 @@ describe("Test btsieve API", () => {
                     })
                     .then(res => {
                         res.should.have.status(201);
-                        location = res.headers.location;
+                        location = res.header.location;
                         location.should.be.a("string");
```

Comment:
🤡 one more.  Maybe check for not empty or a concrete location?
```suggestion
                        location.should.not.be.empty;
```